lib/validate_partitions.pm

```diff
@@ -0,0 +1,53 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+package validate_partitions;
+use Exporter 'import';
+our @EXPORT = qw(validate_partition_table);
+
+use strict;
+use warnings;
+use scheduler 'get_test_suite_data';
+use testapi;
+use Test::Assert ':all';
+
+
+
+sub validate_partition_table {
+    my ($args) = @_;
+    my $expected_table_type;
+    my $root_disk;
+
+    if (exists($args->{root_disk})) { $root_disk = $args->{root_disk} }
```

Comment:
post-fix notation is more used in this case to avoid brackets and make it more readable:
`$root_disk = $args->{root_disk} if exists $args->{root_disk};`

Reply:
Ahm, but you don't need it in this case, just assign `my $root_disk = $args->{root_disk}` and then `unless(defined $root_disk)`.